src/api/procedures/addInvestorUniquenessClaim.ts

```diff
@@ -57,6 +70,9 @@ export const addInvestorUniquenessClaim = new Procedure(prepareAddInvestorUnique
   signerPermissions: {
     tokens: [],
     portfolios: [],
-    transactions: [TxTags.identity.AddInvestorUniquenessClaim],
+    transactions: [
+      TxTags.identity.AddInvestorUniquenessClaim,
+      TxTags.identity.AddInvestorUniquenessClaimV2,
+    ],
```

Comment:
This should be refactored into a `getAuthorization` function so that you can use the type of `proof` to determine which transaction you need permissions for